addons/mail/static/src/models/chat_window_manager/chat_window_manager.js

```diff
@@ -134,7 +136,7 @@ function factory(dependencies) {
                 chatWindow.makeActive({ notifyServer: false });
             }
             // Flux specific: notify server of chat window being opened.
-            if (notifyServer) {
+            if (notifyServer && chatWindowWasFolded !== chatWindow.isFolded) {
```

Comment:
A bit weird that `chatWindowWasFolded` can be `undefined`, whereas `chatWindow.isFolded` is always a boolean, but it's not buggy with current logic.